Tasks/AzureIoTEdgeV2/README.md

```diff
@@ -0,0 +1,9 @@
+# Azure IoT Edge task for Azure Pipelines
+## Overview
+This task is used to build, push IoT Edge modules and deploy IoT Edge solutions.
+## Usage
+Please refer this document for detailed guide: [Continuous integration and continuous deployment to Azure IoT Edge](https://docs.microsoft.com/en-us/azure/iot-edge/how-to-ci-cd)
+## Report an issue
+Please follow the new issue template in this repository to create a new issue for this task.
+## Data/Telemetry
+Azure IoT Edge task for Azure Pipelines collects usage data and sends it to Microsoft to help improve our products and services. Read our [privacy statement](https://go.microsoft.com/fwlink/?LinkID=620956) to learn more. If you don’t wish to send usage data to Microsoft, you can change your telemetry settings by setting variable DISABLE_TELEMETRY to true.
```

Comment:
DISABLE_TELEMETRY  variable is not handled by our code?

Reply:
We already support setting `DISABLE_TELEMETRY` previously, it's handled at https://github.com/microsoft/azure-pipelines-tasks/blob/master/Tasks/AzureIoTEdgeV2/index.ts#L30